apps/angular/projection/src/app/ui/card/card.component.ts

```diff
@@ -37,25 +32,18 @@ import { ListItemComponent } from '../list-item/list-item.component';
     </div>
   `,
   standalone: true,
-  imports: [NgIf, NgFor, ListItemComponent],
+  imports: [ListItemComponent, NgTemplateOutlet],
 })
 export class CardComponent {
   @Input() list: any[] | null = null;
-  @Input() type!: CardType;
   @Input() customClass = '';
+  @Output() AddNewRecordEmitter = new EventEmitter<null>();
+  @Input() ListItemTemplate!: TemplateRef<any>;
```

Comment:
you can use contentchild to project template into your child

Reply:
done